static/src/js/app/components/QuickAddVocabModal.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <div>
+    <div
+      class="modal fade"
+      :class="{ show, 'd-block': active }"
+      tabindex="-1"
+      role="dialog"
+    >
+      <div class="modal-dialog" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h5 class="modal-title">Add Word to Personal Vocabulary List</h5>
+            <button
+              type="button"
+              class="close"
+              data-dismiss="modal"
+              aria-label="Close"
+              @click="toggleModal"
+            >
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div class="modal-body">
+            <div class="form-group">
+              <form id="addvocab-form" @submit.prevent="handleSubmit">
+                <label for="FormControlSelect"
+                  >Select Langauge of Your Personal Vocabulary List</label
+                >
+                <select
+                  v-model="vocabularyListId"
+                  class="form-control mb-2"
+                  id="FormControlSelect"
+                  required
+                >
+                  <option
+                    v-for="langItem in personalVocabLangList"
+                    :key="langItem.id"
+                    :value="langItem.id"
+                  >
+                    {{ formatLang(langItem.lang) }}
+                  </option>
+                </select>
+                <div class="d-flex">
+                  <input
+                    class="form-control mt-2"
+                    type="text"
+                    placeholder="headword"
+                    v-model="headword"
+                    required
+                  />
+                  <input
+                    class="form-control ml-2 mt-2"
+                    type="text"
+                    placeholder="gloss"
+                    v-model="gloss"
+                    required
+                  />
```

Comment:
One aspect of accessible forms is that fields need labels (ideally with semantic html). Placeholder attributes can serve that purpose for folks who do not have vision issues, but I don't think they do much good for screen reader application.

Reply:
Thanks Tylor! Will look into semantic html for accessibility and what is available for screen reader compliance.